src/ui/pages/GetStarted/ImportWallet/MnemonicImportView/AddressImportFlow/AddressImportFlow.tsx

```diff
@@ -131,6 +490,19 @@ function AddressImportList({
         </VStack>
         <PageBottom />
       </PageStickyFooter>
+      <SelectMoreWalletsDialog
+        initialValues={values}
+        wallets={wallets}
+        activeWallets={activeWallets}
+        existingAddressesSet={existingAddressesSet}
+        dialogRef={moreWalletsDialogRef}
+        onSubmit={(values) => {
+          const selectedWallets = wallets
```

Comment:
looks like this can be extracted to a function. Cause I see this filtering logic above too :)

Reply:
I thought about this, too

Reply:
It's now copied in Onboarding as well